.maintain/docker/Dockerfile.aarch64-linux-gnu

```diff
@@ -21,9 +21,7 @@ FROM rustembedded/cross:aarch64-unknown-linux-gnu
 # only for aarch64
 
 # change mirrorlist
-RUN curl https://raw.githubusercontent.com/oooldking/script/master/superupdate.sh | bash && \
```

Comment:
Please don't remove this line. 

Will slow down the speed in local.

Just comment it.

Reply:
Okay.

Another file named chn means CHINA?

Reply:
I move local file to Dockerfile.mirror-chn.aarch64-linux-gnu, because in github action my not need change mirro.

Reply:
Yes, it's China

Reply:
It is not allowed using any kind of mirror on CI/CD, since we're not able to determine if the mirror can be trusted. Even if it is maintained by any well-known organization, the mirror still got a chance having some technical issues such as lagging behind which could potentially cause discrepancy of build results and artifacts.

Reply:
yeah, this pr i removed all used Dockerfile mirror, like this reviews code. and back old file to Dockerfile.mirror-chn.xxx